Sources/AppCoinsSDK/BuildConfiguration.swift

```diff
@@ -106,6 +106,15 @@ internal class BuildConfiguration {
         }
     }
     
+    static internal var attributionServiceBaseURL: String {
```

Comment:
Call this mmpServiceBaseURL, we will probably use it for more than just attribution